packages/polly/polly.0.2.2/opam

```diff
@@ -0,0 +1,26 @@
+opam-version: "2.0"
+synopsis: "Bindings for the Linux epoll system call"
+description: """
+Bindings for the Linux epoll system call. The binding avoids
+  most allocation in the event loop by iterating over all file
+  descriptios that are reported as ready ."""
+maintainer: "Christian Lindig <christian.lindig@citrix.com>"
+authors: "Christian Lindig <christian.lindig@citrix.com>"
+license: "MIT"
+homepage: "https://github.com/lindig/polly"
+doc: "https://github.com/lindig/polly"
+bug-reports: "https://github.com/lindig/polly/issues"
+depends: [
+  "dune" {build}
```

Comment:
```suggestion
  "dune" {>= "2.0"}
```

dune should not be tagged with the build tag due to forward compatibility issues, according to the dune-project file you need at least 2.0

Reply:
@lindig can you upstream this change?

Reply:
I will change it here and then change the opam file in the source repository to match it.